TeamCharles/customer-service
# 15-new-employee-menu :: Users can create a new employee

## Status
**READY**

## Testing

[ ] I have added passing tests for all new functionality.

[X] I have confirmed that all test prior to this pull request still PASS.

## Documentation

X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
Added a new action into `Actions/` called `CreateEmployeeAction`

## Resolves Issue Number

https://github.com/TeamCharles/customer-service/issues/30

## Deploy Notes
To test the action, add to `Program.cs` 's `Main()`

```csharp
CreateEmployeeAction.ReadInput();
```

and check your database file to ensure that a new employee was added after running the program.

```sh
git pull --prune
git checkout 15-new-employee-menu
dotnet run
```

## Steps to Test or Reproduce

1. Add `CreateEmployeeAction.ReadInput()` to your `Program.cs`

1. Add `using customer_service.Actions` to the top of `Program.cs`

1. Run the program

1. Add a new employee

1. Check your database to confirm that the record was created

## Impacted Files in Application

* Actions/CreateEmployeeAction.cs
